Kickstarter-iOS/Assets.xcassets/icons/fb-logo-white.imageset/Contents.json

```diff
@@ -2,14 +2,20 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "fb-logo-white.pdf"
+      "filename" : "fb-logo-white.pdf",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
```

Comment:
See the 'todo' item in the description. I'd like to pair on getting rid of the scale—it was set to 'all' before.